### PR TITLE
#165180332 Create Navbar component

### PR DESCRIPTION
**Pull request by JumaKahiga**
## Description ##
This pull request creates a Navbar component that can be reused in other features; this is necessary to avoid styling conflicts in the final product. 

## Type of change ##
Please select the relevant option
- [ ] Bug fix(a non-breaking change which fixes an issue)
- [x] New feature(a non-breaking change which adds functionality)
- [ ] Breaking change(fix of feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested? ##
Please describe the tests that you ran to verify your changes
- [ ] End to end test
- [x] Integration test
- [ ] unit test

## Checklist: ##
- [x] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes

## Related Pivotal Tracker stories ##
[165180332](https://www.pivotaltracker.com/n/projects/2313279/stories/165180332)

## Reviewers ##
- @andela/olympian-devs
- @mnswaleh 
- @kelvinndmo 
- @Derrickkip 
Screenshot 2019-04-09 at 10 48 15

### CODE REVIEW

**Review by Kibetchirchir — APPROVED** [review]

File: src/components/Navbar/__snapshots__/navbar.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,87 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (Derrickkip) [inline src/components/Navbar/__snapshots__/navbar.test.js.snap:1]: Snapshot folder should be ignored

File: src/css/bootstrap.min.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/*!
```
Comment (Derrickkip) [inline src/css/bootstrap.min.css:1]: Given that we have installed react bootstrap do we need this file?

**Review by winniekariuki — APPROVED** [review]

**Review by winniekariuki — COMMENTED** [review]
Ignore the snapshot folder as the technical team lead has instructed

File: src/css/bootstrap.min.css (hunk 2 above)
Comment (JumaKahiga) [reply src/css/bootstrap.min.css:1]: Yes, we need it for the styling we had agreed for the website.

**Review by Derrickkip — APPROVED** [review]

**Pull request merged**